frontend/src/components/common/password-input/password-input.tsx

```diff
@@ -0,0 +1,48 @@
+import hideIcon from 'assets/icons/hidden-password.svg';
+import showIcon from 'assets/icons/shown-password.svg';
+import { FC, FormControl, FormControlPath } from 'common/types/types';
+import { Image } from 'components/common/common';
+import { getValidClasses } from 'helpers/helpers';
+import { useFormControl, useState } from 'hooks/hooks';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  control: FormControl;
+  name: FormControlPath;
+  inputClassName: string;
+  placeholder?: string;
+};
+
+const PasswordInput: FC<Props> = ({
```

Comment:
Thinking of reusable `InputWithIcon` component, coaches, are we going to have the similar inputs with icons/buttons in other components?

Reply:
good point, but it think for now it can be just a `PasswordInput`